RIOT-OS/RIOT
# drivers/lm75: add SAUL integration

### Contribution description

This hooks up the `lm75` driver to SAUL


### Testing procedure

Add e.g. `USEMODULE += lm75a`  and run `examples/default`.
The lm75 device should show up in the `saul` command. 


### Issues/PRs references
